misc/landing-page/src/components/Examples.astro

```diff
@@ -47,5 +49,27 @@ import evalExample from "../assets/eval.png";
         that this time, every cell is a process – brought to life with Popcorn.
       </Fragment>
     </ExampleCard>
+    <ExampleCard
+      n="03"
+      title="Local Thermostat"
+      href="/demos/local-thermostat"
+      image={localThermostat}
+    >
+      <Fragment slot="description">
+        An example application that uses LocalLiveView.
+        LocalLiveView is a project that aims to store, update and render the assigns similarly to Phoenix LiveView but 
+        using Popcorn API.
```

Comment:
It doesn't really tell how LocalLiveView relates to Popcorn. I'd suggest something like "An example showing how Popcorn integrates with Phoenix LiveView. It allows using live components fully in frontend code, without need for separate server storing components' state.